packages/metadata-common/package.json

```diff
@@ -35,10 +35,10 @@
   "dependencies": {
     "@elyra/services": "3.7.0-dev",
     "@elyra/ui-components": "3.7.0-dev",
-    "@jupyterlab/application": "^3.0.0",
-    "@jupyterlab/apputils": "^3.0.0",
-    "@jupyterlab/codeeditor": "^3.0.0",
-    "@jupyterlab/ui-components": "^3.0.0",
+    "@jupyterlab/application": "^3.3.0-rc.0",
+    "@jupyterlab/apputils": "^3.3.0-rc.0",
+    "@jupyterlab/codeeditor": "^3.3.0-rc.0",
+    "@jupyterlab/ui-components": "^3.3.0-rc.0",
     "@lumino/algorithm": "^1.3.3",
     "@lumino/messaging": "^1.4.3",
     "@lumino/widgets": "^1.17.0",
```

Comment:
Widgets needs to bum to 1.19
https://github.com/jupyterlab/jupyterlab/blob/b71275f8b67a662441eacfba3def4312beeae1f4/packages/fileeditor/package.json#L52

Reply:
widgets is still resolving to 1.17 via yarn.lock. need to check what dep is resolving to that

Reply:
I believe you might need to do a yarn or yarn update from root to update lock?